### PR TITLE
Allow options to be configured from command line

### PR DESCRIPTION
**Pull request by AMecea**

### CODE REVIEW

File: hack/boilerplate.go.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2018 Pressinfra SRL.
+Copyright 2019 Pressinfra SRL.
```
Comment (vtemian) [inline hack/boilerplate.go.txt]: Is it necessary?

File: pkg/cmd/options/options.go [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package options
+
+import "github.com/spf13/pflag"
+
+var (
+	// GitCloneImage used for init container that clone the code
```
Comment (vtemian) [inline pkg/cmd/options/options.go]:
```suggestions
	// GitCloneImage is the image used by the init container that clones the code.
```

File: pkg/cmd/options/options.go [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+	GitCloneImage = "docker.io/library/buildpack-deps:stretch-scm"
+
+	// RcloneImage used for serving uploads from cloud storage
```
Comment (vtemian) [inline pkg/cmd/options/options.go]:
```suggestions
	// RcloneImage is the image used for writing and deleting media files, from cloud storages.
```

File: pkg/cmd/options/options.go [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+	RcloneImage = "quay.io/presslabs/rclone@sha256:4436a1e2d471236eafac605b24a66f5f18910b6f9cde505db065506208f73f96"
+
+	// WordpressRuntimeImage the base image used to run your code
```
Comment (vtemian) [inline pkg/cmd/options/options.go]:
```suggestions
	// WordpressRuntimeImage is the base image used to run your code
```

File: pkg/cmd/options/options.go [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+	WordpressRuntimeImage = "quay.io/presslabs/wordpress-runtime"
+	// WordpressRuntimeTag the tag used for image specifying Wordpress version and php version
```
Comment (vtemian) [inline pkg/cmd/options/options.go]:
```suggestions
	// WordpressRuntimeTag represents the tag of the WordpressRuntimeImage
```

File: pkg/cmd/options/options.go [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+	WordpressRuntimeTag = "5.2-7.3.4-r164"
+)
+
+// AddToFlagSet set command line arguments
+func AddToFlagSet(flag *pflag.FlagSet) {
+	flag.StringVar(&GitCloneImage, "git-clone-image", GitCloneImage, "Set the image to use when cloning code from git.")
```
Comment (vtemian) [inline pkg/cmd/options/options.go]:
```suggestions
	flag.StringVar(&GitCloneImage, "git-clone-image", GitCloneImage, "The image used when cloning from git.")
```

File: pkg/cmd/options/options.go [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+	flag.StringVar(&RcloneImage, "rclone-image", RcloneImage, "Set the image to use for serving images with rclone.")
```
Comment (vtemian) [inline pkg/cmd/options/options.go]:
```suggestions
flag.StringVar(&RcloneImage, "rclone-image", RcloneImage, "The image used by rclone for managing media files.")
```

File: hack/boilerplate.go.txt (hunk 1 above)
Comment (AMecea) [reply hack/boilerplate.go.txt]: Why not?

Comment (vtemian) [reply hack/boilerplate.go.txt]: https://softwareengineering.stackexchange.com/questions/130478/what-copyright-date-for-an-update-to-an-open-source-project-from-last-year

**Pull request merged**